db/models.py

```diff
@@ -14,3 +14,39 @@
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="movies")
+    genres = models.ManyToManyField(Genre, related_name="movies")
```

Comment:
Similarly, the `related_name` for the `genres` field is 'movies'. A more descriptive name like 'genre_movies' could enhance clarity and maintain consistency.